lambo

```diff
@@ -69,7 +69,9 @@ AUTH=false
 NODE=false
 CODEEDITOR=""
 BROWSER=""
-LINK=false' > ~/.lambo/config
+LINK=false
+MYSQL_USERNAME="homestead"
```

Comment:
Definitely a good idea, my only question is do you think we should allow users to specify different credentials for different database drivers? Or is that solving a problem nobody has?